azure-pipelines.yml

```diff
@@ -19,22 +19,23 @@ variables:
   sgx_rust_tag: '2004-1.1.5'
   lcp_tag: '$(Build.SourceVersion)'
 
-
-steps:
-- script: |
-    echo $(GH_PAT) | docker login $(registry) -u $(registry_user) --password-stdin
-    docker pull $(registry)/$(registry_user)/$(sgx_rust_repo):$(sgx_rust_tag)
-  displayName: 'Pull the sgx-rust image'
-
-- script: |
-    docker build -t lcp-ci-it:$(lcp_tag) -f .github/tests/integration/Dockerfile .
-  displayName: 'Build image'
-
-- script: |
-    docker run \
-      --device /dev/sgx/enclave \
-      --device /dev/sgx/provision \
-      --env IAS_KEY=$(IAS_KEY) \
-      --env SPID=$(SPID) \
-      lcp-ci-it:$(lcp_tag)
-  displayName: 'Run integration test'
+jobs:
+  - job: 'IntegrationTest'
```

Comment:
I prefer 'Integration Test` if possible

Reply:
Due to a Pipeline restriction, space characters could not be specified in the job name.

https://learn.microsoft.com/en-us/azure/devops/organizations/settings/naming-restrictions?view=azure-devops#azure-pipelines

Reply:
ok, then let's use `integration_test` instead